gradle/tests.gradle

```diff
@@ -110,7 +110,7 @@ tasks.withType(Test).configureEach {
     showSkippedStandardStreams false
 
     // set to false to hide failed standard out and error streams
-    showFailedStandardStreams true
+    showFailedStandardStreams false
```

Comment:
Why do you want to set this to false?

Reply:
This was when I kept having the CI pipeline being cancelled without explanation so in an attempt to save memory I was removing the logs in the meantime.